README.md

```diff
@@ -115,7 +115,7 @@ The `docs` folder contains a lot more information, including
 [build instructions](docs/building.md), and
 [internal developer docs](docs/development.md).
 
-Refer to the README of each language directory in the `api` folder:
+Refer to the README of each language directory in the `API` folder:
```

Comment:
I think the lower-case spelling is correct as here `api` refers to a folder name, not the abbreviation.